beacon-chain/rpc/endpoints.go

```diff
@@ -1087,5 +1088,14 @@ func (*Service) prysmValidatorEndpoints(coreService *core.Service) []endpoint {
 			handler: server.GetValidatorPerformance,
 			methods: []string{http.MethodPost},
 		},
+		{
+			template: "/prysm/v1/validators/active_set_changes",
+			name:     namespace + ".GetValidatorActiveSetChanges",
```

Comment:
One small nitpick: since the namespace is `prysm.validator`, there is no need to repeat the word `Validator` in every handler. Can you remove it from this one and all the other existing ones?